company/product-vision.md

```diff
@@ -0,0 +1,57 @@
+# Product Vision
+
+## Background
+
+Our [Company Strategy](stratigy.md) sets out 4 pillars that our product work is focussed around
+
+- Provide the best Node-RED experience.
+- Enable users to run Node-RED at the edge.
+- Enable teams to collaborate when working with Node-RED.
+- Meet the compliance needs of the enterprise.
+
+Therefore our product activities should be related back to at least one of these 4 pillars.
+
+IN addition to these pillars we will still do the work needed to ensure a stable, manageable,  platform with a high quality codebase.
+
+## 2022 Plan
+
+Until the end of 2022 We will focus on the following:
+
+In order to **provide the Best Node-RED experience**;
+- We will continue to expose the most useful configuration options for Node-RED within your  project, features like Time Zone support, modifying the editor path and Environment Variables mean that people can use Node-RED on FlowForge in more use-cases
+- Custom Domains for projects will also improve the Node-RED experience allowing users to publish their applications to the web with their own identity.
+- Persistent context will alow users to build flows with better confidence in the integrity of their data.
+- Additional service like Message Brokers and Persistent Storage will provide additional utility and value to the platform while providing the opportunity to increase revenue through add-ons
+- We will work to identify a sustainable solution for building interfaces as part of FlowForge, either through the Node-RED Dashboard and its successor or via another service.
```

Comment:
Potentially we can add something to allow inter project flows. Leveraging our RBAC patterns, mapping to organisations.

Reply:
That would probabbly be done with the additional services like Message Brokers, or as part of the Edge work